src/build.d

```diff
@@ -163,8 +165,13 @@ Command-line parameters
                 lockFile.close();
             }
         }
-        foreach (target; targets.parallel(1))
+
+        taskPool = new TaskPool(jobs - 1); // Main thread is active too
```

Comment:
`max(jobs - 1, 1)` maybe ? In case someone would use -j1 as arg.

Reply:
A `TaskPool`s containing 0 threads is valid and executes every submitted task in the calling thread. `-j1` results in a sequential build.

Reply:
We might need to handle the case where they specify `-j0`.

Reply:
When I use `-j0` with this branch, I get memory allocation failed.

Reply:
Done, `jobs <= 0` will abort the build with an error message.